buildchain/buildchain/image.py

```diff
@@ -82,7 +83,7 @@ def show() -> str:
 
     task = targets.Target(task_dep=['_image_pull', '_image_build']).basic_task
```

Comment:
Would need a dependency on `hardlink` then, no?

```suggestion
     task = targets.Target(
         task_dep=['check_for:hardlink', '_image_pull', '_image_build']
     ).basic_task
```